docker_explorer/container.py

```diff
@@ -161,6 +165,35 @@ def __init__(self, docker_directory, container_id, docker_version=2):
 
     self.log_path = container_info_dict.get('LogPath', None)
 
+  def _GetConfigValue(
+      self, configuration, key, default_value=None,
+      ignore_container_config=False):
+    """Returns the value of a configuration key in the parsed container file.
+
+    Args:
+      configuration(dict): the parsed state from the config.json file.
+      key(str): the key we need the value from.
+      default_value(object): what to return if the key can't be found.
+      ignore_container_config(bool): whether or not to ignore the container's
+        specific configuration (from the ContainerConfig) key.
+
+    Returns:
+      object: the extracted value.
+    """
+    image_config = configuration.get('Config', None)
+    if not image_config:
+      return default_value
+    config_value = image_config.get(key, default_value)
+
+    if not ignore_container_config:
+      # If ContainerConfig has a different value for that key, return this one.
+      container_config = configuration.get('ContainerConfig', None)
+      if container_config:
+        if key in container_config:
+          return container_config.get(key, default_value)
+
+    return config_value
```

Comment:
```suggestion

    if not ignore_container_config:
      # If ContainerConfig has a different value for that key, return this one.
      container_config = configuration.get('ContainerConfig', None)
      if container_config:
        if key in container_config:
          return container_config.get(key, default_value)

    return image_config.get(key, default_value)
```
since config_value isn't used except for return